routes/cube_routes.js

```diff
@@ -3121,6 +3122,30 @@ router.delete('/format/remove/:cubeid/:index', ensureAuth, param('index').toInt(
   }
 });
 
+router.post('/defaultdraftformat/:id', ensureAuth, async (req, res) => {
+  try {
+    const cubeid = req.params.id.split(';')[0];
```

Comment:
I prefer to let the variables be set in the route: e.g.
`:cubeId/defaultdraftFormat/:formatId`